indexer/docs/api/configuration.md

```diff
@@ -0,0 +1,54 @@
+## Configuration
+
+Configuration of the indexer is done with a .toml file
+
+- To configure a known chain such as Optimism, Base, or BaseGoerli simply pass in a `preset = CHAIN_ID`.
+- For custom chains additional configuration is needed to configure such as l1 and l2 contract addresses
```

Comment:
do you plan to have instructions on running / deploying in the docs? feels like a crucial part of the flow that i'd want on first read!

Reply:
Saying it's docker combined with the toml file configuration makes it somewhat self explanatory.   We could include an example docker-compose file like growthbook does https://docs.growthbook.io/self-host#installation